opensearch-project/OpenSearch-Dashboards
# [Workspace] Fix fit&finish for workspace overview page

### Description

- [x] wording change for workspace overview page
- [x] getting started card typography using h4 with s size
- [x] show recently access & updated assets only for current workspace
- [x] hide workspace name in recently assets card when in a workspace
- [x] Use doc link service to get links 
- [x] Recently updated assets use find to search by update_at instead of use recently accessed

### Issues Resolved




## Screenshot



https://github.com/user-attachments/assets/8a508da9-d497-47df-b2cc-b415a731076f



## Testing the changes



1. enable workspace and new home in opensearch_dashboards.yml file
```yml
workspace.enabled: true
uiSettings:
  overrides:
    "home:useNewHomePage": true
```
2. start OSD with `yarn start --no-base-path`
3. create workspace with essential use case
4. verify content in workspace overview page

## Changelog


- fix: fix fit & finish for workspace overview getting started and assets section

### Check List
